test/e2e/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/migration/AbstractMigrationE2EIT.java

```diff
@@ -50,16 +50,10 @@ public abstract class AbstractMigrationE2EIT extends PipelineBaseE2EIT {
     public AbstractMigrationE2EIT(final PipelineTestParameter testParam) {
         super(testParam);
         migrationDistSQLCommand = JAXB.unmarshal(Objects.requireNonNull(PipelineBaseE2EIT.class.getClassLoader().getResource("env/common/migration-command.xml")), MigrationDistSQLCommand.class);
-        if (PipelineEnvTypeEnum.NATIVE == ENV.getItEnvType()) {
-            try {
-                cleanUpPipelineJobs();
-            } catch (final SQLException ex) {
-                throw new RuntimeException(ex);
-            }
-        }
     }
     
-    private void cleanUpPipelineJobs() throws SQLException {
+    @Override
+    protected void cleanUpPipelineJobs() throws SQLException {
         List<String> jobIds = listJobId();
         for (String each : jobIds) {
```

Comment:
Not only support `rollback`, but also add `commit` in case if job is not finished